crates/xtask/src/main.rs

```diff
@@ -215,6 +215,13 @@ struct RunnerOptions {
     #[clap(long)]
     web: bool,
 
+    /// Host to start the webserver.
+    #[clap(long, default_value = "127.0.0.1")]
+    web_host: String,
+    /// Port to start the webserver.
+    #[clap(long, default_value = "5000")]
+    web_port: usize,
```

Comment:
Those flags could actually be optional to avoid duplicating the default value. I'm taking this into account for another suggestion below. Also a port only needs to be a `u16`.

```suggestion
    #[clap(long)]
    web_port: Option<u16>,
```